pkg/aspect/config/quotas.proto

```diff
@@ -32,7 +33,15 @@ message QuotasParams {
         // Map of quota_descriptor label name to attribute expression. At run time each
         // expression will be evaluated to determine the value provided to the aspect. The
         // result of evaluating the expression must match the ValueType of the label in the quota_descriptor.
-        map<string, string> labels = 3;
+        map<string, string> labels = 2;
+
+        // The upper limit for this quota.
+        int64 max_amount = 3;
```

Comment:
I'm still confused on what this field is conveying.

Is this the limit that should be applied for this <scope,subject> (and subject to some effective quota calculation across all <scope,subject>s that apply)? 

Or is this saying that the upper bounds on any limit at this <scope,subject> is X (though there may be lower limits at this <scope,subject>)?

Put another way, is this like `billable_daily_max` or is it just `billable_daily`?